Nika Nabakhteveli

nNabakhteveli opened the pull request:
My resume

nNabakhteveli committed:
Nika Nabakhteveli Resume

My resume

otarza reviewed (changes requested):
@nNabakhteveli  ნიკა გამარჯობა,

ერთი შეხედვით ნორმალურია თუმცა ვიზუალურად დალაგება სჭირდება.

თუ გინდა რომ რეზიუმე უფრო ლამაზი და დალაგებულ იყოს Latex - ში გადაყვანა სცადე. აქ არის ინსტრუქცია: https://github.com/sdolidze/latex-resume

რომ გადაიყვან, ახალი ვარიანტი დააკომიტე და მომწერე აქვე რომ კიდევ შევამოწმო.

:rocket:

nNabakhteveli committed:
Nika Nabakhteveli Commit

გამარჯობა, ოთარ. როგორც მირჩიე, თავიდან ავაწყე ლატექსში რეზიუმე და ეს თემფლეითი ავარჩიე. ფონტი იმდენად არმომწონს, მაგრამ მგონი მთლიანობაში არცისე ცუდია.

nNabakhteveli committed:
Add files via upload

გამარჯობა ოთო. მადლობა რჩევისთვის, და დავამატე კიდევ ერთი პროექტი.

nNabakhteveli committed:
Added one more project

Random Quote Machine

nNabakhteveli committed:
Delete Nika_Nabakhteveli_Resume (1).pdf

nNabakhteveli committed:
Update README.md

nNabakhteveli committed:
Update README.md

nNabakhteveli committed:
Delete 2020Autumn Nika_Nabakhteveli.pdf

nNabakhteveli committed:
Delete old resume

nNabakhteveli committed:
Changed language info

nNabakhteveli committed:
Delete Nika_Nabakhteveli_Resume (2).pdf

nNabakhteveli committed:
Changed language info and fixed typos

nNabakhteveli committed:
Delete Nika_Nabakhteveli_Resume.pdf

nNabakhteveli committed:
LinkedIn link update
